README.md

```diff
@@ -57,91 +67,40 @@ import com.smartsheet.api.oauth.*;
-                                    .setType(ColumnType.TEXT_NUMBER)
-                                    .build();
-
-    // Add the 2 Columns (flag & text) to a new Sheet Object
-    Sheet sheet = new Sheet.CreateSheetBuilder()
-                            .setName("New Sheet")
-                            .setColumns(Arrays.asList(checkboxColumn, textColumn))
-                            .build();
-        
-    // Send the request to create the sheet @ Smartsheet
-    sheet = smartsheet.sheetResources().createSheet(sheet);
-}
-
-/**
- * This provides an example of how to use OAuth to generate a Token from a third party application. It handles
- * requesting the authorization code, sending the user to a specific website to request access and then getting
- * the access token to use for all future requests.
- */
-public static void OAuthExample() throws SmartsheetException, UnsupportedEncodingException, URISyntaxException,
-        NoSuchAlgorithmException {
-
-    // Setup the information that is necessary to request an authorization code
-    OAuthFlow oauth = new OAuthFlowBuilder()
-                            .setClientId("YOUR_CLIENT_ID")
-                            .setClientSecret("YOUR_CLIENT_SECRET")
-                            .setRedirectURL("https://YOUR_DOMAIN.com/").build();
-
-    // Create the URL that the user will go to grant authorization to the application
-    String url = oauth.newAuthorizationURL(EnumSet.of(com.smartsheet.api.oauth.AccessScope.CREATE_SHEETS,
-            com.smartsheet.api.oauth.AccessScope.WRITE_SHEETS), "key=YOUR_VALUE");
-
-    // Take the user to the following URL
-    System.out.println(url);
-
-    // After the user accepts or declines the authorization they are taken to the redirect URL. The URL of the page
-    // the user is taken to can be used to generate an authorization Result object.
-    String authorizationResponseURL = "https://yourDomain.com/?code=l4csislal82qi5h&expires_in=239550&state=key%3D12344";
-
-    // On this page pass in the full URL of the page to create an authorizationResult object
-    AuthorizationResult authResult = oauth.extractAuthorizationResult(authorizationResponseURL);
-
-    // Get the token from the authorization result
-    Token token = oauth.obtainNewToken(authResult);
-
-    // Save the token or use it.
-}
+// Set the Access Token
+Token token = new Token();
+token.setAccessToken("ll352u9jujauoqz4gstvsae05");
+
+// You can List Sheets with or without pagination. Here are samples for both ways to do it.
+
+// Sample 1: Omit 'include' parameter and pagination parameters
+PagedResult<Sheet> sheets = smartsheet.sheetResources().listSheets(
+    null,           // EnumSet<SourceInclusion> includes
+    null,           // PaginationParameters
+    null            // Date modifiedSince
+);
+
+// Sample 2: Specify pagination parameter 'includeAll' 
+PaginationParameters parameters = new PaginationParameters.PaginationParametersBuilder().setIncludeAll(true).build();
```

Comment:
I wouldn't show pagination here. Not required and covered in the docs.

Please copy the C# example and use the first sheetId returned as the sheet to get. (Leave a commented line to allow the user to override.)